resources.nix

```diff
@@ -15,72 +15,76 @@ let
     inherit pkgs inputs;
   };
 
+  ibc-packages = import ./resources/ibc-go {
+    inherit pkgs inputs;
+  };
+
   # Cosmos packages
-  packages = rec {
+  packages = with inputs; rec {
```

Comment:
I consider the `with` statement as being an [antipattern](https://nix.dev/anti-patterns/language#with-attrset-expression) that should be avoided. It makes the source code difficult to parse to find out which variable is coming from where.

Reply:
I think its fine when its localized and the context is kindof implicit. But I am happy to revert

Reply:
Would be good to revert. The scope here is pretty large, and reader have to look quite far ahead to find out the with statement.